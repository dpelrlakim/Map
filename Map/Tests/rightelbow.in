[] a 0
[] a 5
[] a 9
[] a 8
[] a 7
[] a 6
[] a 4
[] a 3
[] a 2
erase a 9
erase a 8
erase a 7
erase a 5
erase a 6
erase a 4
erase a 3
erase a 0
[] a -3
[] a -2
[] a -1
erase a -3
erase a -2
erase a 2
erase a -1
clear a
