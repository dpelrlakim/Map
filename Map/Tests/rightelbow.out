0
a: (0,0)
0
a: (0,0) (5,0)
0
a: (0,0) (5,0) (9,0)
0
a: (0,0) (5,0) (8,0) (9,0)
0
a: (0,0) (5,0) (7,0) (8,0) (9,0)
0
a: (0,0) (5,0) (6,0) (7,0) (8,0) (9,0)
0
a: (0,0) (4,0) (5,0) (6,0) (7,0) (8,0) (9,0)
0
a: (0,0) (3,0) (4,0) (5,0) (6,0) (7,0) (8,0) (9,0)
0
a: (0,0) (2,0) (3,0) (4,0) (5,0) (6,0) (7,0) (8,0) (9,0)
a: (0,0) (2,0) (3,0) (4,0) (5,0) (6,0) (7,0) (8,0)
a: (0,0) (2,0) (3,0) (4,0) (5,0) (6,0) (7,0)
a: (0,0) (2,0) (3,0) (4,0) (5,0) (6,0)
a: (0,0) (2,0) (3,0) (4,0) (6,0)
a: (0,0) (2,0) (3,0) (4,0)
a: (0,0) (2,0) (3,0)
a: (0,0) (2,0)
a: (2,0)
0
a: (-3,0) (2,0)
0
a: (-3,0) (-2,0) (2,0)
0
a: (-3,0) (-2,0) (-1,0) (2,0)
a: (-2,0) (-1,0) (2,0)
a: (-1,0) (2,0)
a: (-1,0)
a:
a:
